### PR TITLE
Documents the community_edition flag

### PR DESCRIPTION
**Pull request by ypc-faros**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ There are two ways that arguments can be passed into the script. The first, is v
 
 | Argument                         | Description                                                                                                                              | Required | Default                     |
 | -------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- | -------- | --------------------------- |
-| &#x2011;&#x2011;api_key          | Your Faros API key. See the documentation for more information on [obtaining an api key](https://docs.faros.ai/#/api?id=getting-access). | Yes      |                             |
+| &#x2011;&#x2011;api_key          | Your Faros API key. See the documentation for more information on [obtaining an api key](https://docs.faros.ai/#/api?id=getting-access). | Yes (unless the `community_edition` flag is present)      |                             |
```
Comment (tovbinm) [inline README.md:90]: better 
```
not required for [Community Edition](https://github.com/faros-ai/faros-community-edition), i.e when `community_edition` flag is present
```

File: README.md [hunk 2]
```diff
@@ -97,6 +97,8 @@ There are two ways that arguments can be passed into the script. The first, is v
 | &#x2011;&#x2011;skip_saving_run  | Do not include cicd_Build in the event. (no value accepted, true if flag is present)                                                     |          | False                       |
 | &#x2011;&#x2011;no_lowercase_vcs | Do not lowercase commit_organization and commit_repo. (no value accepted, true if flag is present)                                       |          | False                       |
 | &#x2011;&#x2011;validate_only    | Event will not be consumed but instead will only be validated against event schema. (no value accepted, true if flag is present)         |          | False                       |
+| &#x2011;&#x2011;community_edition| Mutations derived from events will be sent to local Hasura. (no value accepted, true if flag is present)         |          | False                       |
```
Comment (tovbinm) [inline README.md:100]: No need to disclose the internals. perhaps better:
better 
```
| &#x2011;&#x2011;community_edition| Events will be formatted and sent to [Faros Community Edition](https://github.com/faros-ai/faros-community-edition). (no value accepted, true if flag is present) | | False |
```

File: README.md [hunk 3]
```diff
@@ -141,6 +143,20 @@ A `CD` event communicates the outcome of an application deployment pipeline exec
 | &#x2011;&#x2011;run_status_details    | Any extra details about the status of the job run executing the deployment.                                                                                   |                                                           |                                                                |
 | &#x2011;&#x2011;run_start_time        | The start time of the job run in milliseconds since the epoch. (e.g. `1626804346019`)                                                                         |                                                           |                                                                |
 | &#x2011;&#x2011;run_end_time          | The end time of the job run in milliseconds since the epoch. (e.g. `1626804346019`)                                                                           |                                                           |                                                                |
+### Usage with Faros Community Edition
```
Comment (tovbinm) [inline README.md:146]: great!

File: README.md [hunk 4]
```diff
@@ -141,6 +143,20 @@ A `CD` event communicates the outcome of an application deployment pipeline exec
...
+
+When using Faros Community Edition, you can use the tool in exactly the same way as described above. Just include the `community_edition` flag. The Faros API key is not needed, since the tool will call your locally deployed Hasura to perform mutations derived from the events. See the [Faros Community Edition repo](https://github.com/faros-ai/faros-community-edition) for more details.
+
+```sh
+$ ./faros_event.sh CI \
```
Comment (tovbinm) [inline README.md:151]: If I run the Faros Community Edition not on a localhost and if I specify `--url https://my-faros.com:9999` would it work?

File: README.md (hunk 1 above)
Comment (ypc-faros) [reply README.md]: Done 👍

File: README.md (hunk 4 above)
Comment (ypc-faros) [reply README.md:151]: Done in a separate PR 👍

Comment (tovbinm) [reply README.md:151]: let's put a `--`url` argument in this example with localhost + port where the Hasura is

**Review by tovbinm — APPROVED** [review]

**Pull request merged**